lib/plugins/aws/deploy/compile/events/apiGateway/lib/method/integration.js

```diff
@@ -146,7 +146,7 @@ module.exports = {
         #if( $keyValSize >= 1 )
           #set( $key = $util.escapeJavaScript($util.urlDecode($keyVal[0])) )
           #if( $keyValSize >= 2 )
```

Comment:
Yeah, this part is a bit confusing because we have JSON enveloped inside a x-form-urlencoded body, this is splitting the key from the value of the form-urlencoded content first, from what I could tell. We shouldn't need to strip on the key, since i'm pretty sure single quotes aren't allowed there.  The VTL here is a little confusing, I'm basing my assumptions on the $key/$val variables and how they're used further down.